pkg/server/server.go

```diff
@@ -421,6 +423,16 @@ func (s *Server) HTTPHandler() http.Handler {
 		)
 	}
 
+	clusterManagementProxy := proxy.NewProxy(s.ClusterManagementProxyConfig)
+	handle(accountManagementEndpoint, http.StripPrefix(
+		s.BaseURL.Path,
+		authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
+			accessTokenHeader := r.Header.Get("AccessToken")
```

Comment:
If the client sent the token in the `Authorization` request header instead of `AccessToken`, I don't think we'd need any special handling of the header here.

Reply:
✔️